### PR TITLE
Summary report and aggregation service landings

### PR DESCRIPTION
**Pull request by heyawhite**
New pages for summary reports and the aggregation service for ARA/PAA, plus redirects and URL updates.

- [Aggregation service](https://deploy-preview-4421--developer-chrome-com.netlify.app/docs/privacy-sandbox/aggregation-service/)
- [Summary reports](https://deploy-preview-4421--developer-chrome-com.netlify.app/docs/privacy-sandbox/summary-reports/)

Larger updates:
- [ARA overview](https://deploy-preview-4421--developer-chrome-com.netlify.app/docs/privacy-sandbox/attribution-reporting/)

### CODE REVIEW

File: site/en/docs/privacy-sandbox/attribution-reporting/index.md
Comment (heyawhite) [inline site/en/docs/privacy-sandbox/attribution-reporting/index.md:391]:
```suggestion
- Event-level reports associate 64 bits of information on the ad side
  (`news.example`) with only 1 bit or 3 bits on the conversion side
  (`shop.example`). 64 bits are enough information to be mapped to an
  individual user identifier, but these 64 bits can only be linked with very
  little cross-site information: 1 bit or 3 bits, which are not enough to hold
  an identifier.
  - The ad-side 64 bits are not new information. A user ID can already be
    available on the ad side today. `news.example` or `adtech.example` 
    already knows about a certain user's activity on `news.example`.
```

**Review by maudnals — APPROVED** [review]
LGTM!
Do you think we could link the Handbook in the PS landing page (https://deploy-preview-4421--developer-chrome-com.netlify.app/docs/privacy-sandbox/), as part of the changes you made in this PR?
Handbook: https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#

**Comment by heyawhite** [comment]
> LGTM! Do you think we could link the Handbook in the PS landing page (https://deploy-preview-4421--developer-chrome-com.netlify.app/docs/privacy-sandbox/), as part of the changes you made in this PR? Handbook: https://docs.google.com/document/d/1BXchEk-UMgcr2fpjfXrQ3D8VhTR-COGYS1cwK_nyLfg/edit#

Sure thing -- I've added it and will wait for preview confirmation before merging

**Pull request merged**
